identity/src/main/java/com/android/identity/mdoc/request/DeviceRequestParser.java

```diff
@@ -16,6 +16,8 @@
 
 package com.android.identity.mdoc.request;
 
+import static java.rmi.server.LogStream.log;
```

Comment:
nit: I have removed this unused import and unnecessary extra line below -- this file is essentially reverted.